db.js

```diff
@@ -461,6 +461,10 @@ module.exports = function (log, error) {
           account.lockedAt = null
           delete accountUnlockCodes[uid]
           return {}
+        },
+        function(err) {
+          // the only error from this.account(uid) could be a 404 Not Found - mask this
```

Comment:
It may be worth commenting on *why* we're masking it, just for future readers.  e.g. "it's harmless and simpler to just ignore it than force error handling downstream".